api-definitions/README.md

```diff
@@ -0,0 +1,23 @@
+# API Definitions
+
+The common abstract interface for the service APIs is defined by [REST APIs](https://www.redhat.com/en/topics/api/what-is-a-rest-api) using [OpenAPI](https://www.openapis.org/). This project is the central home for these API definitions.
+
+## Style Guidelines
+
+* Use `kebab-case` for path elements
+* Use `snake_case` for properties
+* Use `UpperCamelCase` for internal reusable schema names
```

Comment:
Would it be helpful to provide links to the case definitions?

Reply:
I was hoping the example usage in backticks would be sufficient. Do you think we need further explanation than that? External links seem overkill to me, but I could expand on the definition. Maybe one point to make is that for path elements in particular, we need to use [ascii](https://www.w3schools.com/tags/ref_urlencode.ASP) to avoid percent encoding.

Reply:
Updated with more detailed guidelines for each